packages/block-library/src/list/v2/edit.js

```diff
@@ -165,7 +172,7 @@ function Edit( { attributes, setAttributes, clientId } ) {
 		<>
 			<TagName
 				reversed={ reversed }
-				start={ start }
+				start={ Platform.isWeb ? start : undefined }
```

Comment:
Passing `start` to the `View` component on mobile breaks the styling of the list

Reply:
Does that only happen for `ul`, or for both?

Reply:
Might be good to have a separate file for this tag name for native and web.

Reply:
>Does that only happen for ul, or for both?

It happens for both.

> Might be good to have a separate file for this tag name for native and web.

That's a good idea! I'll update it 👍

Reply:
Hey @ellatrix 👋 I've applied the changes in https://github.com/WordPress/gutenberg/pull/42702/commits/8ccdd3532b752085dc3ff3a50cc0f7ef86c90a4b let me know what you think. Thanks!